xmr-btc/src/bitcoin/timelocks.rs

```diff
@@ -0,0 +1,48 @@
+use serde::{Deserialize, Serialize};
+use std::ops::Add;
+
+/// Represent a timelock, expressed in relative block height.
```

Comment:
🙃 Might be nice to mention the BIP as reference: https://github.com/bitcoin/bips/blob/master/bip-0068.mediawiki